qa/tasks/lost_unfound.py

```diff
@@ -7,14 +7,15 @@
 from tasks.util.rados import rados
 from teuthology import misc as teuthology
 from teuthology.orchestra import run
+import json
```

Comment:
./tasks/lost_unfound.py:10:1: F401 'json' imported but unused
ERROR: InvocationError for command /home/jenkins-build/build/workspace/ceph-pull-requests/qa/.tox/flake8/bin/flake8 --select=F,E9 --exclude=venv,.tox (exited with code 1)